### PR TITLE
Changes so far

### PR DESCRIPTION
**Pull request by GrissomE**

### CODE REVIEW

File: Day_15/Coffee_Machine.py [hunk 1]
```diff
@@ -1,102 +1,78 @@
 MENU = {
     "espresso": {
         "ingredients": {
-            "water": 50,
-            "coffee": 18,
+            "Water": 50,
+            "Coffee": 18,
```
Comment (aswerve97) [inline Day_15/Coffee_Machine.py:5]: Who cares?

File: Day_15/Coffee_Machine.py [hunk 2]
```diff
@@ -1,102 +1,78 @@
...
         },
-        "cost": 1.5,
+        "Cost": 1.5,
```
Comment (aswerve97) [inline Day_15/Coffee_Machine.py:7]: ? Are you good?

**Review by aswerve97 — CHANGES_REQUESTED** [review]

File: Day_15/Coffee_Machine.py (hunk 2 above)
Comment (GrissomE) [reply Day_15/Coffee_Machine.py:7]: Very

**Review by aswerve97 — APPROVED** [review]
I will steal this code and use it to get hired XD

**Pull request merged**